source/monitoring/cluster-monitoring.txt

```diff
@@ -0,0 +1,130 @@
+.. _c-cluster-monitoring:
+
+==================
+Cluster Monitoring
+==================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, server, topology
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecols
+
+Overview
+--------
+
+This guide shows you how to use the {+driver-short+} to monitor server
+discovery and monitoring (SDAM) events in a MongoDB instance, replica
+set, or sharded cluster. These events occur when there are any changes
+in the state of the MongoDB instance or cluster that you are connected
+to.
+
+You might use information about SDAM events in your application to
+understand cluster changes, assess cluster health, or perform capacity
+planning.
+
+.. _c-subscribe-sdam:
+
+Subscribe to Events
+-------------------
+
+You can access details about SDAM events by subscribing to them
+in your application. To subscribe to an event, create a value of
+type ``mongoc_apm_callbacks_t`` and create an event callback function
+for the event you want to subscribe to. Use the callback's corresponding
+setter method to set the callback on the ``mongoc_apm_callbacks_t`` and 
+pass this type to the ``mongoc_client_set_apm_callbacks()`` function.
```

Comment:
Recommended rewording:

> To subscribe to an event, define a callback function to handle each event type you want to subscribe to. Use a `mongoc_apm_callbacks_t` object to register the list of APM callbacks with a client object using `mongoc_client_set_apm_callbacks()`.

There is a lot of overlap with the actual code example below. Wordy details can be delegated to the code example.

Also recommend referring to these user-defined functions as "APM callbacks" or "APM callback functions" for consistency with C Driver API documentation.